OnBaseDocsApi/Controllers/OnBaseDocsController.cs

```diff
@@ -108,22 +294,31 @@ IHttpActionResult DocumentResult(Document doc)
 
             return Ok(new DataResult<DocumentAttributes>
             {
-                Data = new DocumentAttributes
+                Data = new DataResource<DocumentAttributes>
                 {
-                    ID = doc.ID,
-                    CreatedBy = doc.CreatedBy.ID,
-                    DateStored = doc.DateStored,
-                    DocumentDate = doc.DocumentDate,
-                    Status = doc.Status.ToString(),
-                    Name = doc.Name,
-                    DocumentType = doc.DocumentType.Name,
-                    DefaultFileType = doc.DefaultFileType.Name,
-                    LatestAllowedRevisionID = doc.LatestAllowedRevisionID,
-                    Keywords = keywords,
+                    Id = doc.ID,
+                    Type = "onbaseDocument",
+                    Attributes = new DocumentAttributes
+                    {
+                        ID = doc.ID,
```

Comment:
Since we moved the ID to the top level, I think we can remove it from the attributes

Reply:
Done.